Source/DiabloUI/settingsmenu.cpp

```diff
@@ -73,16 +74,24 @@ void GoBackOneMenuLevel()
 	shownMenu = ShownMenuType::Settings;
 }
 
+void UpdateDescription(OptionEntryBase *pOption)
+{
+	auto paragraphs = WordWrapString(pOption->GetDescription(), rectDescription.size.width, GameFont12, 1);
+	CopyUtf8(optionDescription, paragraphs, sizeof(optionDescription));
+}

```

Comment:
This function can take `const OptionEntryBase &option` as the argument. Polymorphic calls still work on references even if it was using a virtual method.

Reply:
Changed 🙂